cep-7.md

```diff
@@ -0,0 +1,33 @@
+<table>
+<tr><td> Title </td><td> Conda Capitalization Standard </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Katherine Kinnaman &lt;kkinnaman@anaconda.com&gt;</td></tr>
+<tr><td> Created </td><td> Jun 28, 2022</td></tr>
+<tr><td> Updated </td><td> July 5, 2022</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+The capitalization conventions for conda need to be decided and written down within the conda docs so that they can be enforced during PRs, for consistency within written materials about and related to conda.
+
+## Specification
+
+Conda should be lowercase except when used at the beginning of a sentence. When speaking about the command, code formatting should be used (i.e. `conda`)
```

Comment:
And what about the rare oddball cases when the sentence starts with the command? E.g.:

> `conda create ...` will create a new environment.

Otherwise, it would simply be incorrect:

> `Conda create ...` will not create a new environment.

Are we simply disallowing `conda` commands from occurring at the beginning or do we allow this exception?

Reply:
Great edge case to consider. I think in that case, we would not have the capitalization apply to the command.

So maybe a rewrite of "Conda should be written in lowercase. References to the `conda` command should use code formatting. If the use of conda is not a command and if conda is at the beginning of a sentence, conda should be uppercase."

Reply:
Are we only referring to conda as the tool? Or also extended meanings like "the conda ecosystem", "conda packages", "conda channels"...?

Reply:
Yeah, I want to make this as simple to follow as possible, so yes, it would extend. Basically, anything that isn't a reference to the command itself should just be regular text.